package.json

```diff
@@ -18,21 +18,21 @@
     "@react-navigation/native": "^5.9.4",
     "@react-navigation/stack": "^5.14.4",
     "envy": "^2.0.0",
-    "expo": "^41.0.0",
+    "expo": "^42.0.0",
     "expo-app-loading": "^1.0.3",
-    "expo-font": "~9.1.0",
-    "expo-status-bar": "^1.0.4",
+    "expo-font": "~9.2.1",
```

Comment:
Are we using the tilde because these are big/finicky dependencies?

Reply:
Expo CLI did this, and I trust it more than I trust me!

Reply:
Ah, got it!  Yes, happy to pass off that job to expo :P